packages/cli/src/playground/src/App.tsx

```diff
@@ -52,7 +53,7 @@ function App() {
                 <Route path="/networks" element={<Networks />} />
                 <Route
                   path="/networks/v-next/:networkId"
-                  element={<Navigate to="/networks/v-next/:networkId/chat" />}
+                  element={<NavigateTo to="/networks/v-next/:networkId/chat" />}
```

Comment:
Don't you want to do this instead?

```suggestion
                  loader: ({ params }) => {
                    return redirect(`/networks/v-next/${params.networkId}/chat`, 301)
                  },
                  element={null}
```